ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1795,6 +1954,39 @@ objects:
         elif [[ "${CLUSTER_TYPE}" == "vsphere" ]]; then
           terraform apply -auto-approve -var 'bootstrap_complete=true' -no-color &
           wait "$!"
+        elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
+          az network nsg rule delete -g $RESOURCE_GROUP --nsg-name ${INFRA_ID}-controlplane-nsg --name bootstrap_ssh_in
+          az vm stop -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap
+          az vm deallocate -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap
+          az vm delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap --yes
+          az disk delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap_OSDisk --no-wait --yes
+          az network nic delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap-nic --no-wait
+          az storage blob delete --account-key $ACCOUNT_KEY --account-name $ACCOUNT_NAME --container-name files --name bootstrap.ign
+          az network public-ip delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap-ssh-pip
+
+          export KUBECONFIG=/tmp/artifacts/installer/auth/kubeconfig
+
+          public_ip_router="<pending>"
+          while [[ $public_ip_router =~ "pending" ]]
+          do
+            public_ip_router=`oc -n openshift-ingress get service router-default --no-headers | awk '{print $4}'`
+            echo $public_ip_router
+          done
+          
+          az network dns record-set a add-record -g $BASE_DOMAIN_RESOURCE_GROUP -z ${base_domain} -n *.apps.${CLUSTER_NAME} -a $public_ip_router --ttl 300
+          
+          az network private-dns record-set a create -g $RESOURCE_GROUP -z ${CLUSTER_NAME}.${base_domain} -n *.apps --ttl 300
+          az network private-dns record-set a add-record -g $RESOURCE_GROUP -z ${CLUSTER_NAME}.${base_domain} -n *.apps -a $public_ip_router
+
+          echo "Deploying 06_workers"
```

Comment:
Probably want to deploy workers before bootstrap-complete. It used to be that the default router would not propagate until there were some compute nodes. I think the "compact cluster" work has possible removed that requirement. Even so, by deploying compute nodes right after masters (instead of waiting several minutes for bootstrap complete) you reduce the overall time it takes the cluster to reach "install-complete".

Reply:
Right, fixed.